docs/downloads/automation-library/integrations/javadoc/review_javadoc_changes.cm

```diff
@@ -0,0 +1,21 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+automations:
+  # Assign PRs that only affect JavaDocs to the technical writing team and add docs label
+  review_javadoc:
+    if:
+      - {{ source.diff.files | matchDiffLines(regex=r/\/*\*([\s\S]*?)\//) }}
```

Comment:
following `matchDiffLines` there should be an aggregation function missing, i.e. missing `| every }}` or `| some }}`

Reply:
Good catch. Fixed!